types/helpers/helpers.color.d.ts

```diff
@@ -8,6 +10,8 @@ export function color(
   | [number, number, number, number]
 ): ColorModel;
 
+export function isPatternOrGradient(value: AnyObject): boolean;
```

Comment:
I think `Color` would suit the value better (string|CanvasGradient|CanvasPattern)

Reply:
I thought about that, but since it checks for objects in general and then if it is a pattern or gradient I thought this would be more in line with what you can input. 

Although we only use it in the color part and I don't think you would logically try to feed this random objects.

Reply:
It should allow string too, so string|AnyObject then?